smoke-tests/src/test/java/org/hypertrace/agent/smoketest/SpringBootSmokeTest.java

```diff
@@ -148,20 +148,4 @@ public void get() throws IOException, InterruptedException {
     //    Assertions.assertEquals(1, responseBodyAttributes.size());
     //    Assertions.assertEquals("Hi!", responseBodyAttributes.get(0));
   }
-
-  @Test
-  public void blocking() throws IOException {
```

Comment:
this is obviously a loss. To make this test work with mock filer not being in the final distribution we will have to build test java agent distro - this might be related to https://github.com/hypertrace/javaagent/issues/238